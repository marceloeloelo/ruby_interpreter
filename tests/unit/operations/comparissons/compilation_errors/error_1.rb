# TypeError: Fixnum can't be coerced into String
if ("foo" > 1)
	puts("hola")
end
