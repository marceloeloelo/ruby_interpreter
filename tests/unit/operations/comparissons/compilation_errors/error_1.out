TypeError: Fixnum can't be coerced into String
